pkg/postage/postagecontract/contract_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package postagecontract_test
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	postageMock "github.com/ethersphere/bee/pkg/postage/mock"
+	"github.com/ethersphere/bee/pkg/postage/postagecontract"
+	"github.com/ethersphere/bee/pkg/settlement/swap/transaction"
+	transactionMock "github.com/ethersphere/bee/pkg/settlement/swap/transaction/mock"
```

Comment:
time to move transaction pkg out of swap

Reply:
agreed, but let's please do code moves in a separate PR after this